pkg/rpcclient/rolemgmt/roles_test.go

```diff
@@ -149,3 +152,45 @@ func TestDesignateAsRoleTransaction(t *testing.T) {
 		require.Equal(t, ta.tx, tx)
 	}
 }
+
+func TestDesignationEvent_FromStackItem(t *testing.T) {
+	tests := []struct {
+		name        string
+		items       []stackitem.Item
+		expectedErr error
+		expected    DesignationEvent
+	}{
+		{
+			name:        "role overflow",
+			items:       []stackitem.Item{stackitem.NewBigInteger(big.NewInt(256)), stackitem.NewBigInteger(big.NewInt(1))},
```

Comment:
```suggestion
			items:       []stackitem.Item{stackitem.Make(256), stackitem.Make(1)},
```

Shorter line, more readable code.